### ISSUE TITLE
Open Issues with Alexandria Map

### ISSUE THREAD

**Issue by akariv** [akariv/atlas-medliq#3]
https://medliq.art/alexandria/

- [ ] Contact Us & Credits pages are empty
- [ ] Site is not responsive - looks very small on mobile screens
- [ ] To integrate in main page (https://medliq.art), we need:
   - Full official title of the map
   - Short description of the map
   - Map preview images for mobile and desktop. See examples and required image sizes from here: 
     - https://github.com/akariv/atlas-medliq/blob/master/src/assets/img/map-thumbnail-m1-desktop.png
     - https://github.com/akariv/atlas-medliq/blob/master/src/assets/img/map-thumbnail-m1-mobile.png

**Comment by aemaddin** [comment]
Hello @akariv,

Map title: Water/Past, present and future

Description: An artistic map of Alexandria representing ancient cistern systems and their evolution and the challenges facing drinking water till present times.

Image: ![](https://raw.githubusercontent.com/akariv/atlas-medliq/master/src/assets/img/social-m7.png)

**Comment by aemaddin** [comment]
<img width="224" alt="map-thumbnail-m7-mobile" src="https://user-images.githubusercontent.com/11630742/195658751-dcd2c2ad-09cf-465d-b789-84ba612e6aef.png">

**Comment by aemaddin** [comment]
<img width="224" alt="map-thumbnail-m7-mobile" src="https://user-images.githubusercontent.com/11630742/195659340-0bd4b484-b0cf-4592-8dad-4404370c9c7c.png">

for mobile

**Comment by akariv** [comment]
Thanks, don't you think though that the map's title should at least mention
Alexandria?

**Comment by aemaddin** [comment]
Yes, it's a valid point. Just let me check with the owners and revert to you.

**Comment by aemaddin** [comment]
Hello @akariv

We may use "Ancient cisterns of Alexandria" as a title.

**Comment by akariv** [comment]
Great - done.

https://medliq.art/
